src/azure-cli-core/azure/cli/core/extension/operations.py

```diff
@@ -53,6 +56,8 @@ def _run_pip(pip_exec_args):
     except CalledProcessError as e:
         logger.debug(e.output)
         logger.debug(e)
+        if "PermissionError: [WinError 5]" in e.output:
```

Comment:
what's default install path after the change? suppose no change

Reply:
yes. no change for the default extension path.